src/index.js

```diff
@@ -209,15 +215,22 @@ export function makeUseAxios(configurationOptions) {
       useAxios.__ssrPromises.push(axiosInstance(config))
     }
 
-    const cancelOutstandingRequest = React.useCallback(() => {
+    const cancelOutstandingRequest = React.useCallback(message => {
       if (cancelSourceRef.current) {
-        cancelSourceRef.current.cancel()
+        cancelSourceRef.current.cancel({
```

Comment:
are you sure about the signature of the cancel method? based on what I read in the [axios source](https://github.com/axios/axios/blob/master/index.d.ts#L122) it accepts an optional string argument, not an object

Reply:
It might be smart to follow that convention, but I can say that it definitely works on newer Chrome, logs it similar to a `console.error`.